net/dnsproxy/files/dnsproxy.config

```diff
@@ -0,0 +1,42 @@
+
+# For documents, please see https://github.com/AdguardTeam/dnsproxy#usage
+
+config dnsproxy 'enabled'
+	option enabled '0'
```

Comment:
splitting this seems excessive, maybe rename `config` to `global` and move this there?

Reply:
Done.